### PR TITLE
Adds, registers Team model to Web app

### PR DESCRIPTION
**Pull request by yadavankit**
@deshraj please review :)

### CODE REVIEW

File: apps/web/admin.py [hunk 1]
```diff
@@ -2,11 +2,18 @@
...
     list_display = ("name", "email", "message",)
     list_filter = ("email",)
     search_fields = ("email",)
+
+
+@admin.register(Team)
+class TeamAdmin(admin.ModelAdmin):
+    list_filter = ("team_type", "email",)
+    list_display = ("name", "email", "team_type",)
+    search_fields = ("email",)
```
Comment (deshraj) [inline apps/web/admin.py]: Please add a new line here.

Comment (deshraj) [inline apps/web/admin.py]: Also, add `name` in search fields.

File: apps/web/migrations/0002_team.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```
Comment (deshraj) [inline apps/web/migrations/0002_team.py]: Can you delete this migration file and try creating migrations using this command with name
`python manage.py migrate --settings=settings.dev --name=added_team_model`

File: apps/web/models.py [hunk 3]
```diff
@@ -17,3 +17,33 @@ def __unicode__(self):
...
+    email = models.EmailField(max_length=70,)
+    headshot = models.ImageField(upload_to="headshots", null=True, blank=True)
+    visible = models.BooleanField(default=True)
+    github_url = models.URLField(max_length=200,)
+    linkedin_url = models.URLField(max_length=200,)
+    personal_website = models.URLField(max_length=200,)
+    team_type = models.CharField(choices=TEAM_TYPE_OPTIONS, max_length=50,)
+
+    def __unicode__(self):
+        return "%s: %s: %s:" % (self.name, self.email, self.team_type)
```
Comment (deshraj) [inline apps/web/models.py]: Please return `self.name`

File: frontend/src/js/controllers/authCtrl.js [hunk 4]
```diff
@@ -56,6 +56,9 @@
 
             //reset error msg
             vm.wrnMsg = {};
+
+            //switch off form errors
+            vm.isFormError = false;
```
Comment (deshraj) [inline frontend/src/js/controllers/authCtrl.js]: Please remove this file as this is not the part of PR.

File: apps/web/models.py [hunk 5]
```diff
@@ -17,3 +17,33 @@ def __unicode__(self):
...
+    # Team Type Options
+    CORE_TEAM = 'Core Team'
+    CONTRIBUTOR = 'Contributor'
+
+    TEAM_TYPE_OPTIONS = (
+        (CORE_TEAM, CORE_TEAM),
+        (CONTRIBUTOR, CONTRIBUTOR),
+    )
+
+    name = models.CharField(max_length=100,)
```
Comment (TroJan) [inline apps/web/models.py]: can you please remove the trailing comma after 100. Infact please remove it for every other field (email, github_url,...)

File: frontend/src/js/controllers/authCtrl.js [hunk 6]
```diff
@@ -56,6 +56,9 @@
 
             //reset error msg
             vm.wrnMsg = {};
+
```
Comment (TroJan) [inline frontend/src/js/controllers/authCtrl.js]: can you please explain why is this here ?

File: apps/web/admin.py [hunk 7]
```diff
@@ -2,11 +2,18 @@
...
 @admin.register(Contact)
 class ContactAdmin(TimeStampedAdmin):
     list_display = ("name", "email", "message",)
     list_filter = ("email",)
     search_fields = ("email",)
+
+
+@admin.register(Team)
+class TeamAdmin(admin.ModelAdmin):
+    list_filter = ("team_type", "email",)
```
Comment (TroJan) [inline apps/web/admin.py]: can you please remove `email` from list_filter. It actually makes no sense.

File: apps/web/admin.py (hunk 1 above)
Comment (TroJan) [inline apps/web/admin.py]: please add a new line here

File: apps/web/migrations/0002_team.py (hunk 2 above)
Comment (yadavankit) [inline apps/web/migrations/0002_team.py]: Ok sure 👍

File: frontend/src/js/controllers/authCtrl.js (hunk 6 above)
Comment (yadavankit) [inline frontend/src/js/controllers/authCtrl.js]: @TroJan yeah...will be removing this file

File: apps/web/models.py [hunk 8]
```diff
@@ -7,13 +7,42 @@
 
 class Contact(TimeStampedModel):
     """Model representing details of User submitting queries."""
-    name = models.CharField(max_length=100,)
-    email = models.EmailField(max_length=70,)
-    message = models.CharField(max_length=500,)
+    name = models.CharField(max_length=100)
+    email = models.EmailField(max_length=70)
+    message = models.CharField(max_length=500)
```
Comment (deshraj) [inline apps/web/models.py]: @yadavankit these three lines are not part of this issue. Please remove these.

Comment (yadavankit) [inline apps/web/models.py]: @deshraj done 👍
